main.tf

```diff
@@ -102,6 +102,12 @@ resource "aws_eks_cluster" "this" {
     aws_cloudwatch_log_group.this,
     aws_iam_policy.cni_ipv6_policy,
   ]
+
+  lifecycle {
+    ignore_changes = [
+      access_config["bootstrap_cluster_creator_admin_permissions"]
```

Comment:
```suggestion
      access_config.bootstrap_cluster_creator_admin_permissions
```

Reply:
Since access_config is a block, bootstrap_cluster_creator_admin_permissions must be referenced using the index, not an attribute.

I assumed your suggested syntax initially as well, however it results in the following error. 
```
Block type "access_config" is represented by a list of objects, so it must be indexed using a numeric key, like .access_config[0].
```

Reply:
oh, weird. ok, will go with what you've got then!